MatrixKit/Models/MXKAppSettings.h

```diff
@@ -252,6 +252,16 @@ typedef NS_ENUM(NSUInteger, MXKKeyPreSharingStrategy)
  */
 @property (nonatomic, getter=isCallKitEnabled) BOOL enableCallKit;
 
+#pragma mark - Spaces
+
+/**
+ Return YES if the user wants to display all rooms in home
+ 
+ This boolean value is defined in shared settings object with the key: `enableShowAllRoomsInHome`.
+ Return NO if no value is defined.
+ */
+@property (nonatomic, getter=isShowAllRoomsInHomeEnabled) BOOL enableShowAllRoomsInHome;
```

Comment:
Could this become `enableShowAllRoomsInHomeSpace` so as to not cause any confusion with the Home tab?

Personally I'd happily drop the `enable`(`d`) part too so it's like the `showLeftMembersInRoomMemberList` setting but that's just my own preferences there 🙂